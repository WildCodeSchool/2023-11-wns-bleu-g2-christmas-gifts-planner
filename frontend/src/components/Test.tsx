export default function Test(props: { hello: string }) {
  const { hello } = props;
  return (
    <div className="w-[400px]">
      <h1 className="text-2xl font-bold">Test component</h1>
      <p>{hello}</p>
    </div>
  );
}
