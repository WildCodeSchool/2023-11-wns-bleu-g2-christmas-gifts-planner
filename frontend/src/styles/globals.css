@tailwind base;
@tailwind components;
@tailwind utilities;

* {
  box-sizing: border-box;
}

html,
body {
  height: 100dvh;
}

#__next {
  height: 100%;
}

.genericButton {
  border: solid 1px transparent;
  border-radius: 6px;
  padding: 6px;
  transition: all 0.2s;
}
.genericButton:hover {
  background-color: #f0f0f0;
}
