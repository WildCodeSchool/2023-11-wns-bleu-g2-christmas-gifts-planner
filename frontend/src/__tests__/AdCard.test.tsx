import { describe, it, expect } from "@jest/globals";
import { render, screen } from "@testing-library/react";
import Test from "../components/Test";

describe("Test component", () => {
    it("renders correctly", () => {
        const view = render (
        <Test 
        hello="Hello World" />);
        expect(view.baseElement).toMatchSnapshot();
    });
});