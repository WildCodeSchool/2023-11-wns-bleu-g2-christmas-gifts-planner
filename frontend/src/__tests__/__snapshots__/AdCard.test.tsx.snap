// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component renders correctly 1`] = `
<body>
  <div>
    <div
      class="w-[400px]"
    >
      <h1
        class="text-2xl font-bold"
      >
        Test component
      </h1>
      <p>
        Hello World
      </p>
    </div>
  </div>
</body>
`;
